lms/djangoapps/grades/tests/test_new.py

```diff
@@ -239,56 +240,155 @@ def test_save_and_load(self):
 
 
 @ddt.ddt
-class TestMultipleProblemTypesSubsectionScores(ModuleStoreTestCase, ProblemSubmissionTestMixin):
+class TestMultipleProblemTypesSubsectionScores(SharedModuleStoreTestCase):
```

Comment:
It wasn't.  Turns out mocking get_score bypassed that altogether.  I moved the mock in further to _get_submissions_score, and tested what happens when that returns None values for possible scores.  It now verifies that the fix was necessary.